Vagrantfile

```diff
@@ -47,6 +47,13 @@ Vagrant.configure(2) do |config|
     vb.cpus = 2
   end
 
+  host_user = ENV.fetch("USER", "vagrant")
+  aws_profile = ENV.fetch("RF_AWS_PROFILE", "raster-foundry")
+  rf_aws_config_bucket = ENV.fetch("RF_AWS_CONFIG_BUCKET",
```

Comment:
I went with `RF_AWS_CONFIG_BUCKET` because that's how we have ours named (rasterfoundry-<env>-config`) and I wanted to keep the environment variable as close to what we have it named to be less confusing.

Maybe I'll try to remove the `RF_SETTINGS_BUCKET` to make it consistent.